packages/extension-sample/src/setup-server.ts

```diff
@@ -4,7 +4,30 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-/* istanbul ignore next */
-export default ({app}) => {
-    console.log('Setup Server App')
-}
\ No newline at end of file
+
+import { IApplicationExtension } from '@salesforce/pwa-kit-runtime/ssr/server/extensibility/types'
```

Comment:
Is there something going on wit your linter? I don't remember named imports having spaces in the curly braces.

Reply:
The extension-sample is only for testing. I plan to remove it before merging the PR.